server/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -404,6 +419,10 @@ private void setEnableQueryPhaseParallelCollection(boolean enableQueryPhaseParal
         this.enableQueryPhaseParallelCollection = enableQueryPhaseParallelCollection;
     }
 
+    private void setMemoryAccountingBufferSize(ByteSizeValue memoryAccountingBufferSize) {
```

Comment:
Just a general comment :D I see this pattern of a single use via method-ref named method for setting listeners. Why not just inline this stuff to a lambda, save 3 lines and let the compiler add the one off method? :)